torch/fx/experimental/GraphManipulation.py

```diff
@@ -0,0 +1,23 @@
+from typing import List
+from torch.fx.graph_module import GraphModule
+from torch.fx.graph import map_arg
+
+
+def get_all_users_of(index: int, fx_module: GraphModule) -> List[int]:
```

Comment:
nit: The module should probably go first. You can imagine this would be a method on `Module`, and conceptually it would make sense for the object this method acts on to come first

Reply:
One thing to watch out for in the future: indices into the `Node` list will become stale if we end up editing the Node list. If written in the functional style as is the case today, this should be fine, but it's something to watch out for in the future